module/NPC-sheet.js

```diff
@@ -418,26 +418,26 @@ async _onDropItem(event, data) {
 
       function createContent(quantity) {
         let maxQuantityText = "";
-        if (maxQuantity != null) maxQuantityText = `&nbsp;&nbsp;of ${maxQuantity}`;
+        if (maxQuantity != null) maxQuantityText = `&nbsp;&nbsp;${game.i18n.localize("CYPHERSYSTEM.Of")} ${maxQuantity}`;
         let content = `<div align="center">
-        <label style='display: inline-block; width: 98px; text-align: right'><b>Quantity/Units: </b></label>
+        <label style='display: inline-block; width: 98px; text-align: right'><b>${game.i18n.localize("CYPHERSYSTEM.Quantity")}/${game.i18n.localize("CYPHERSYSTEM.Units")}: </b></label>
         <input name='quantity' id='quantity' style='width: 75px; margin-left: 5px; margin-bottom: 5px;text-align: center' type='text' value=${quantity} data-dtype='Number'/>` + maxQuantityText + `</div>`;
         return content;
       }
 
       function buttons() {
         if (maxQuantity != null) {
-          return ({move: {icon: '<i class="fas fa-share-square"></i>', label: "Move", callback: (html) => moveItems(html.find('#quantity').val(), itemData)}, moveAll: {icon: '<i class="fas fa-share-square"></i>', label: "Move All", callback: (html) => moveItems(maxQuantity, itemData)}, cancel: {icon: '<i class="fas fa-times"></i>', label: "Cancel", callback: () => { }}})
+          return ({move: {icon: '<i class="fas fa-share-square"></i>', label: game.i18n.localize("CYPHERSYSTEM.Move"), callback: (html) => moveItems(html.find('#quantity').val(), itemData)}, moveAll: {icon: '<i class="fas fa-share-square"></i>', label: game.i18n.localize("CYPHERSYSTEM.MoveAll"), callback: (html) => moveItems(maxQuantity, itemData)}, cancel: {icon: '<i class="fas fa-times"></i>', label: game.i18n.localize("CYPHERSYSTEM.Cancel"), callback: () => { }}})
         } else {
-          return ({move: {icon: '<i class="fas fa-share-square"></i>', label: "Move", callback: (html) => moveItems(html.find('#quantity').val(), itemData)}, cancel: {icon: '<i class="fas fa-times"></i>', label: "Cancel", callback: () => { }}})
+          return ({move: {icon: '<i class="fas fa-share-square"></i>', label: game.i18n.localize("CYPHERSYSTEM.Move"), callback: (html) => moveItems(html.find('#quantity').val(), itemData)}, cancel: {icon: '<i class="fas fa-times"></i>', label: game.i18n.localize("CYPHERSYSTEM.Cancel"), callback: () => { }}})
         }
       }
 
       function moveItems(quantity, itemData) {
         quantity = parseInt(quantity);
         if (item.data.data.quantity != null && (quantity > item.data.data.quantity || quantity <= 0)) {
           moveDialog(quantity, itemData);
-          return ui.notifications.warn(`You can only move between 1 and ${item.data.data.quantity} items.`);
+          return ui.notifications.warn(`${game.i18n.localize("CYPHERSYSTEM.CanOnlyMoveCertainAmountOfItems")} ${item.data.data.quantity} ${game.i18n.localize("CYPHERSYSTEM.Items").toLowerCase()}.`);
```

Comment:
We shouldn’t use `toLowerCase()` here. “Items” is a noun, which some languages always capitalize.

Reply:
Noted

Reply:
Refactored translation